client/src/utils/Store.js

```diff
@@ -362,13 +345,16 @@ export default class Store {
    * Sets the plot type for the plot, and generates a new plot if it is
    * different.
    *
-   * @param {string} newPlotType the identifier of the new plot type. For
+   * @param {string} plotType the identifier of the new plot type. For
    * example: `dndm`.
+   * @param {string} axis the axis to change. For example: `x`.
+   * @param {boolean} refresh if true, gets new plot data
+   * @returns {void}
    */
-  setPlotType = async (newPlotType) => {
-    if (newPlotType !== this.state.plotType) {
-      this.state.plotType = newPlotType;
-      await this.getPlotData();
+  setPlotType = async (plotType, axis, refresh) => {
+    if (plotType !== this.state[axis]) {
+      this.state[axis] = plotType;
+      if (refresh) await this.getPlotData();
     }
   }
 }
```

Comment:
Why is `this.state.axis` not defined in the initialization part of the `Store` if it is used here?

Reply:
axis is a string variable. either "x" or "y". It's written in the docs

Reply:
Oh there it is. Alright cool.